app/assets/javascripts/components/overview/details.jsx

```diff
@@ -127,7 +128,9 @@ const Details = createReactClass({
 
   poll() {
     return setInterval(() => {
-      if (this.state.updateCount > MAX_UPDATE_COUNT) { return; }
+      const documentVisible = document.hidden;
```

Comment:
This is confusing. I would expect `document.hidden` to be true when the page is not focused, but `documentVisible` suggests the opposite. Is this a mistake, or a quirk of `document`? If it's the latter, I suggest using a different constant name and/or a comment to explain how it works.